Art/Luv/index.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
```

Comment:
If you like, you can rename the title. Document is kinda generic. Just a suggestion, you can resolve this comment without changing anything, it's just a suggestion.